app/views/admin/editions/_image_fields_case_studies.html.erb

```diff
@@ -32,7 +32,7 @@
                 <%= image_data_fields.upload :file, required: false, class: 'js-upload-image-input' %>
               <% end %>
               <%= images_fields.text_field :alt_text, label_text: 'Alt text' %>
-              <%= images_fields.text_area :caption, rows: 2 %>
+              <%= images_fields.text_field :caption %>
```

Comment:
Hmmm I'm not sure why this change is necessary. You shouldn't need to change the input type to get something showing or to get a test to pass 😕  It's introduced some conditionals in `test/support/admin_edition_controller_test_helpers.rb ` that adds complexity.

I'd be tempted to drop this commit and find another fix for the issue - would you like to pair on that? Or are there some steps I can follow to reproduce the issue myself?

Reply:
This was a really weird issue. So if the caption field is set to a text_area it doesn't appear when the rest of the image_fields appear.

<img width="764" alt="Screenshot 2020-10-13 at 12 59 55" src="https://user-images.githubusercontent.com/17481621/95857825-1bd09300-0d54-11eb-82f4-e1e6b6415f4a.png">
^^ This screenshot is when the field is set to be a text_area. It's on the page to create a new case_study after clicking on the `Use a custom image` radio button.

The text area method comes from [here](https://github.com/alphagov/whitehall/blob/774eb78cd4c75622e786bdb18762850cd0b91840/lib/whitehall/form_builder.rb#L88-L97), I wasn't sure if it was running in a peculiar order after adding in the javascript, I tried some debugging in this text_area method and in the new showHide function but can't spot anything obvious as to why it isn't showing.

I would love a hand with it if you have some time :)

Reply:
Argh, it looks like a [training feature](https://github.com/alphagov/whitehall/blob/41b611f3d8c092706b3a27f7063b6d014a0efb88/config/initializers/enable_words_to_avoid_highlighting.rb#L7-L9) is interfering with the output of the textarea :( I've had a quick play and can't get this `highlightTextarea` thing to go away (even if I set that const to false). Turning off JavaScript makes the textarea render as it should.

<img width="433" alt="Screenshot 2020-10-13 at 15 40 28" src="https://user-images.githubusercontent.com/5111927/95875914-71179f00-0d6a-11eb-97c2-8de9b02f77fe.png">

I have to leave right now but I'll have another play when I come back - it's worth investigating if that feature is even still being used!

Reply:
I've had another look and found a fix - details in the commit: https://github.com/alphagov/whitehall/commit/7a455d094b27ad966c161f5af7b01123d2330ae1

Reply:
WOOO Awesome!!

Reply:
I've cherry-picked you commit over and made a small change to apply it to the summary text-area as well as the body to get the tests passing

Reply:
Nice - just tested it locally and it works 👍